blog/2023-07-13-reproducibility-designer.md

```diff
@@ -0,0 +1,41 @@
+---
+
+OpenRefine is hiring a designer to help shape our improvements to OpenRefine's operation history and test those changes with user panels.
+<!--truncate-->
+
+[OpenRefine](https://openrefine.org/) is a powerful tool to clean messy data, popular in a diverse range of communities. It has been serving the needs of journalists, librarians, Wikipedians, scientists for more than ten years and is taught in many curricula and workshops around the world. OpenRefine is a fiscally sponsored project of [Code for Science & Society Inc](https://codeforscience.org/), a 501(c)(3) charitable organization in the USA.
+
+We are currently working on improving the reproducibility of OpenRefine workflows as part of [a project funded by the Essential Open Source Software for Science programme](https://chanzuckerberg.com/eoss/proposals/improving-openrefines-reproducibility/).
+This consists in improving the user experience around OpenRefine's operation history, which tracks all transformations applied on the original data. This work aims to address a range of long standing issues, such as:
+- limitations of the Undo/Redo functionality, such as the lack of ability to undo an operation in the middle of the history list ([#183](https://github.com/OpenRefine/OpenRefine/issues/183)) or unintentional loss of work after using the undo feature and
+  applying a new operation ([#369](https://github.com/OpenRefine/OpenRefine/issues/369), [#3184](https://github.com/OpenRefine/OpenRefine/issues/3184))
+- the difficulty in reusing and sharing OpenRefine workflows, due to the lack of error handling when applying series of operations and the difficulty to adapt a workflow to a new project (for instance, with different column names)
+- the lack of support for running OpenRefine workflows as part of larger pipelines, without using the web UI interactively. This covers for instance the lack of import metadata in the JSON export of the history
+  ([#460](https://github.com/OpenRefine/OpenRefine/issues/460)).
+
+See the corresponding [GitHub project](https://github.com/orgs/OpenRefine/projects/6) for more issues in the scope of this project.
+
+We are looking for a part-time designer who will
+* design changes to the user interface to support those improvements;
+* validate those choices by conducting user testing sessions;
+* help prioritize follow-up improvements based on user feedback.
+
+Requirements: 
+* Experience with designing and testing software
+* Ability to work in a fully remote environment
+* Good communication skills and fluency in English
+
+Nice to have:
+* Familiarity with OpenRefine
+* Experience with open source development practices and tools (such as the use of GitHub Issues)
+
+**TODO compensation details**
+This role is available immediately and is initially expected to span the remaining duration of this project around reproducibility (1.5 year). We hope to be able to extend this role to work on other areas of the tool as other opportunities arise.
```

Comment:
A 1 1/2 year initial term sounds like a lot. It's much easier to extend contracts than to cut them short. I'd recommend six month renewable (normally I might suggest 90 days, but I think there'll be a fair amount of learning curve and startup overhead).

Reply:
That makes sense. My motivation with advertising the full grant duration is to make it clear we are aiming to retain the person if everything works out nice. 6 months renewable sounds fine.